docs/software/sciapps/lammps.md

```diff
@@ -1,4 +1,385 @@
+    variable        z index 200
+    variable        t index 1000
+
+    variable        xx equal 1*$x
+    variable        yy equal 1*$y
+    variable        zz equal 1*$z
+
+    variable        interval equal $t/2
+
+    units           lj
+    atom_style      atomic
+
+    lattice         fcc 0.8442
+    region          box block 0 ${xx} 0 ${yy} 0 ${zz}
+    create_box      1 box
+    create_atoms    1 box
+    mass            1 1.0
+
+    velocity        all create 1.44 87287 loop geom
+
+    pair_style      lj/cut 2.5
+    pair_coeff      1 1 1.0 1.0 2.5
+
+    neighbor        0.3 bin
+    neigh_modify    delay 0 every 20 check no
+
+    fix             1 all nve
+
+    thermo          ${interval}
+    thermo_style custom step time  temp press pe ke etotal density
+    run_style       verlet
+    run             $t
+    ```
+
+### Running on Eiger
+
+On Eiger, the following sbatch script can be used:
+
+```bash title="run_lammps_eiger.sh"
+#!/bin/bash -l
+#SBATCH --job-name=<JOB_NAME>
+#SBATCH --time=01:00:00 (1)
+#SBATCH --nodes=2                   
+#SBATCH --ntasks-per-core=1                                                    
+#SBATCH --ntasks-per-node=32 (2)
+#SBATCH --cpus-per-task=4 (3) 
+#SBATCH --account=<ACCOUNT> (4)
+#SBATCH --hint=nomultithread
+#SBATCH --hint=exclusive
+#SBATCH --constraint=mc                                                  
+#SBATCH --uenv=<LAMMPS_UENV>:/user-environment (5)
+#SBATCH --view=kokkos (6)
+
+ulimit -s unlimited
+
+export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK
+
+srun --cpu-bind=socket lmp -k on t $OMP_NUM_THREADS -sf kk -in lj_kokkos.in
+```
+
+1. Time format: `HH:MM:SS`.
+2. Number of MPI ranks per node.
+3. Number of threads per MPI rank.
+4. Change `<ACCOUNT>` to your project account name.
+5. Change `<LAMMPS_UENV>` to the name (or path) of the LAMMPS uenv you want to use.
+6. Enable the `kokkos` uenv view.
+
+Note that the same input file `lj_kokkos.in` can be used as with running LAMMPS with Kokkos on the HPC Platform.
+
+### Building LAMMPS from source
+
+#### Using CMake
+
+If you'd like to rebuild LAMMPS from source to add additional packages or to use your own customized code, you can use the develop views contained within the uenv image to provide you with all the necessary libraries and command-line tools you'll need.
+For the following, we'd recommend obtaining an interactive node and building inside the tmpfs directory.
+
+```bash
+salloc -N1 -t 60 -A <account>
+srun --pty bash
+mkdir /dev/shm/lammps_build; cd /dev/shm/lammps_build
+```
+
+After you've obtained a version of LAMMPS you'd like to build, extract it in the above temporary folder and create a build directory. 
+Load one of the two following views:
+
+```
+#build environment for lammps +kokkos package, without providing lmp executable
+uenv start --view develop-kokkos lammps/2024:v2-rc1
+#build environment for lammps +gpu package, without providing lmp executable
+uenv start --view develop-gpu lammps/2024:v2-rc1
+```
```

Comment:
Same as above, it might be good to use tabs here to distinguish between the two views. I'd also include the CMake command below in eigher tab, with/without the Kokkos-specific variables.